tests/source/structs.rs

```diff
@@ -1,10 +1,31 @@
 #![crate_type = "lib"]
 
-/// A unit struct.
-pub struct UnitStruct;
+// @has /data/relationships/structs/data/0/type 'struct'
+// @has /data/relationships/structs/data/0/id 'structs::UnitStruct'
 
 // @has /included/0/type 'struct'
 // @has /included/0/id 'structs::UnitStruct'
 // @has /included/0/attributes/name 'UnitStruct'
 // @has /included/0/attributes/docs 'A unit struct.'
-// @has /data/relationships/structs/data/0/id 'structs::UnitStruct'
+
+/// A unit struct.
+pub struct UnitStruct;
+
+// @has /data/relationships/structs/data/1/type 'struct'
+// @has /data/relationships/structs/data/1/id 'structs::ContainerStruct'
+
+// @has /included/1/type 'struct'
+// @has /included/1/id 'structs::ContainerStruct'
+// @has /included/1/attributes/name 'ContainerStruct'
+// @has /included/1/attributes/docs 'A struct that contains another struct.'
```

Comment:
As you can see there's no "Docs for the ContainerStruct." here. That part of the documentation just seems to disappear.